evalml/tests/integration_tests/test_data_checks_and_actions_integration.py

```diff
@@ -231,7 +231,7 @@ def test_data_checks_impute_cols(problem_type):
         expected_pipeline_class = MulticlassClassificationPipeline
         y_expected = ww.init_series(
             pd.Series([0, 1, 2, 2, 2]),
-            logical_type="Integer",
+            logical_type="IntegerNullable",
```

Comment:
These testing changes are due to the target imputer changes to maintain original logical type if possible